### ISSUE TITLE
Unable to download

### ISSUE THREAD

**Issue by fizerkhan** [centic9/CommonCrawlDocumentDownload#10]
I got following error when I tried to download. I run this command `./gradlew downloadDocuments`

```
2017-10-30 06:35:53 INFO    [DownloadFromCommonCrawl] Downloading line 1: 0.0000%, having 0 downloaded: {"urlkey": "io,freshsales)/", "timestamp": "2017082...
Exception in thread "main" java.lang.IllegalStateException: Unknown field found: mime-detected
        at org.dstadler.commoncrawl.index.CDXItem.parse(CDXItem.java:60)
        at org.dstadler.commoncrawl.index.DownloadFromCommonCrawl.main(DownloadFromCommonCrawl.java:43)
:downloadDocuments FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':downloadDocuments'.
> Process 'command '/usr/lib/jvm/java-8-oracle/bin/java'' finished with non-zero exit value 1
````

**Comment by centic9** [comment]
You likely are running with the released version of the code, can you try the latest version from git? 

I'll try to release a 1.0.0.5 soon so some recent changes are reflected in the released code as well.

**Comment by fizerkhan** [comment]
Got it. Thanks.

**Comment by centic9** [comment]
I have now released 1.0.0.5 which contains this fix, this should appear on Maven Central in the next few days.
